src/transformers/generation/logits_process.py

```diff
@@ -474,12 +476,43 @@ def _calc_banned_ngram_tokens(
 
 class NoRepeatNGramLogitsProcessor(LogitsProcessor):
     r"""
-    [`LogitsProcessor`] that enforces no repetition of n-grams. See
+    N-grams are groups of "n" consecutive words, characters, or tokens taken from a sequence of text. Given a sentence:
+    " She runs fast ", the bi-grams (n = 2) would be ("she","runs") and ("runs","fast"). In text generation, avoiding
+    repetitions of word sequences provides a more diverse and coherent output. This [`LogitsProcessor`] enforces no
```

Comment:
```suggestion
    repetitions of word sequences provides a more diverse output. This [`LogitsProcessor`] enforces no
```
diverse -- absolutely
coherent -- depends on context, as you mention in your New York example below :)
(as such, removed the coherent)

Reply:
Makes sense. I will make the change.